rel/overlay/etc/default.ini

```diff
@@ -189,27 +210,29 @@ database_prefix = userdb-
 [httpd]
 port = {{backend_port}}
 bind_address = 127.0.0.1
-authentication_handlers = {couch_httpd_auth, cookie_authentication_handler}, {couch_httpd_auth, default_authentication_handler}
-secure_rewrites = true
-allow_jsonp = false
+
 ; Options for the MochiWeb HTTP server.
 ;server_options = [{backlog, 128}, {acceptor_pool_size, 16}]
 ; For more socket options, consult Erlang's module 'inet' man page.
 ;socket_options = [{recbuf, undefined}, {sndbuf, 262144}, {nodelay, true}]
 socket_options = [{sndbuf, 262144}]
-enable_cors = false
-enable_xframe_options = false
+
+; These settings were moved to [chttpd]
+;authentication_handlers =
```

Comment:
We don't have to necessarily list them one their own line with = signs, could probably do a list ensuring it is less than 80 columns per line. Something like:
```
; These settings were moved to [chttpd]
;authentication_handlers, secure_rewrites, allow_jsonp, enable_cors,
;enable_xframe_options, max_uri_length, ...

Reply:
changed